internal/backend/oauth/oauth.go

```diff
@@ -45,6 +46,14 @@ func New(l *zap.Logger) sdkservices.OAuth {
 	if githubBaseURL == "" {
 		githubBaseURL = "https://github.com"
 	}
+	var err error
+	githubBaseURL, err = kittehs.NormalizeURL(githubBaseURL, true)
+	if err != nil {
+		l.Fatal("Invalid environment variable value",
+			zap.String("name", "GITHUB_ENTERPRISE_URL"),
```

Comment:
don't you want to include the value?

Reply:
I can, but the value is already included in the error message (see `strings.go` line 76), and I want to keep it in the error for other callers that return the error and don't print a log line, e.g. `jwt.go`.

Reply:
sg